source/manage/cloud-billing.rst

```diff
@@ -4,10 +4,19 @@ Mattermost Cloud billing
 .. include:: ../_static/badges/allplans-cloud.rst
   :start-after: :nosearch:
 
-Mattermost Cloud is offered as a pay-as-you-go subscription service.
+From February 2023, Mattermost Cloud Professional and Enterprise plans are offered as an annual subscription service.
 
-You will be billed for registered users in your workspace. A registered user is a user who has an account in a workspace and does not show as *Inactive* in **System Console > User Management > Users**. Registered users will count towards your monthly invoice each day as long as they are not deactivated in the billing period.
+.. note:: 
+   If you have an existing Cloud Professional monthly subscription you can keep it as is and are not required to switch to an annual subscription. If you'd like to switch, please follow these steps.
```

Comment:
@johndavidlugtu, are there steps for this and will these customers ever need to switch to annual?

Reply:
There will be a [section in the System Console as well as a dismissible global banner](https://github.com/mattermost/mattermost-webapp/pull/12028) nudging them to switch to annual.

Note that the CTA might change based on this [conversation](https://community.mattermost.com/private-core/pl/rgf441zqw7y6jnonihbkwmehjo) @justinegeffen